service-front/module/Application/view/application/general/home/index.twig

```diff
@@ -13,6 +13,10 @@
 
     <p>We’ll save your LPA as you progress so you do not need to complete it in one go.</p>
 
+    <div class="panel panel-border-narrow govuk-details__text" role="lpa-survey" aria-label="LPA survey">
+        <p><a href="https://eu.surveymonkey.com/r/MLPArecruitment" rel="noreferrer noopener" target="_blank">Complete our short survey<span class="visually-hidden"> (opens in new tab)</span></a> if you’d like to take part in research to help us improve the way LPAs are made.</p>
+    </div>
+ 
```

Comment:
```suggestion
  
```

Reply:
Thanks :) That was a silly space I had to insert to try to wake up github